AIEnrichmentPipeline/functions/EnrichmentPipeline.Functions.Domain/Extensions/Extensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Primitives;
+using EnrichmentPipeline.Functions.Domain.Models;
+using EnrichmentPipeline.Functions.Domain.Constants;
+
+namespace EnrichmentPipeline.Functions.Domain.Extensions
+{
+    /// <summary>
+    /// A location to define extension methods.
+    /// </summary>
+    public static class Extensions
```

Comment:
[Optional] It is easier if you have ILoggerExtensions class and ScopeDataExtensions class separately